### PR TITLE
Issue 495 visualization from graph ml

### LINKED ISSUE #495: Create visualizations from GraphML files
The GraphML files have vertex locations and caller regions (in latitude and longitude); the information should be sufficient to generate a pixelated cartogram image. See Alex Neary's capstone materials in Google Drive: Readings/Lab Publications/Internal.

Likely, this will just be Python code in the Tools subdirectory.

### PR DESCRIPTION
**Pull request by silvercube227**
resolves #495 

Created a visualization of King County 911 system from GraphML data. Color coded nodes for ease of use.

### CODE REVIEW

File: Tools/gis2graph/__pycache__/random.cpython-310.pyc
Comment (stiber) [inline Tools/gis2graph/__pycache__/random.cpython-310.pyc:1]: Delete this from the repo, add `__pycache__` to the `.gitignore`, and tell git to untrack this stuff.

File: Tools/gis2graph/visualize_graphML.py
Comment (stiber) [inline Tools/gis2graph/visualize_graphML.py:1]: Move to `Tools/graphMLVis`

**Review by stiber — CHANGES_REQUESTED** [review]
This branch is off of `master`, but should be off of `development`. Figure some good way to get this to merge into `development`.

File: Tools/gis2graph/__pycache__/random.cpython-310.pyc
Comment (whiteja2) [reply Tools/gis2graph/__pycache__/random.cpython-310.pyc:1]: __pycache__ has been removed, added to the .gitignore, and untracked.

File: Tools/gis2graph/visualize_graphML.py
Comment (whiteja2) [reply Tools/gis2graph/visualize_graphML.py:1]: visualize_graphML.py has been moved to the graphMLVis directory.

**Review by stiber — APPROVED** [review]

**Pull request merged**